src/domain-services/categories/model.ts

```diff
@@ -0,0 +1,35 @@
+import { type Op } from '@unocha/hpc-api-core/src/db/util/conditions';
+
+export type CategoryGroup =
```

Comment:
We also should use types from `hpc-api-core` here.

Reply:
Since its not used anywhere in the code - I'll be removed
In the future if needed, as you suggest, we should import from `hpc-api-core`.